src/main/java/edu/mum/sonet/advisor/ExceptionAdvice.java

```diff
@@ -49,6 +49,7 @@ public DomainErrors handleException(Exception exception) {
         DomainErrors errors = new DomainErrors();
         errors.setErrorType("error");
         errors.setMessage(exception.getMessage());
+        exception.printStackTrace();
```

Comment:
Printing the stack when handling errors isn't a good idea. I'm guessing you added it for debugging purposes.

Reply:
yes it was for debugging to catch the problem